mu-plugins/central-hub/tests/phpunit/integration/config-store/getAllKeys.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Tests for the API function getAllKeys().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Integration\ConfigStore
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Integration\ConfigStore;
+
+use function KnowTheCode\ConfigStore\_the_store;
+use function KnowTheCode\ConfigStore\loadConfig;
+use function KnowTheCode\ConfigStore\getAllKeys;
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+/**
+ * Class Tests_GetAllKeys
+ *
+ * @package spiralWebDb\centralHub\Tests\Integration\ConfigStore
+ * @group   config-store
+ */
+class Tests_GetAllKeys extends Test_Case {
+
+	/**
+	 * Empty the store before starting these tests.
+	 */
+	public static function setUpBeforeClass() {
```

Comment:
@rgadon107 The task of emptying the store shouldn't have to be part of the 1st test.  Rather, the 1st test should assume the environment is ready for it to run its tests.  Therefore, we need a way to empty the store before the tests start running in this class.  PHPUnit gives us a static method called `setUpBeforeClass`.  It runs before the tests and only runs once (vs. the `setUp` method which runs for each test).

Notice how I pulled "empty the store" into this method.  Does it make sense to you?

Reply:
@hellofromtonya With each test I write, I'm always learning something new; whether it's a previously unknown (to me) static method, a PHP code construct, or an application of Brain Monkey. That's cool that PHPUnit provides that static method `setUpBeforeClass`. Didn't know about it until just now. Went an looked it up in the PHPUnit documentation to learn a bit more about it. 

What you did makes sense.

Reply:
Awesome.  When you need to pre-setup a test class, you can use the `setUpBeforeClass`.  If you need to set up tests that run with each test method in the class, then you use `setUp`.

* `setUpBeforeClass`: runs before any of the test methods in the class and only runs when the class loads
* `setUp`: runs before each test method in the class.
* `tearDown`: runs after each test method in the class.
* `tearDownAfterClass`: runs after all of the test methods run in the class and only runs once.

Let's break that down to make sure you get.  There are 2 integration tests in this PR, i.e. 2 test methods.  Here would be execution order:

* Class loads
* `setUpBeforeClass` runs
* `setUp` runs
* test method 1 `test_should_return_all_store_keys` runs
* `tearDown` runs
* `setUp` runs
* test method 2 `test_should_return_empty_array_when_store_is_empty ` runs
* `tearDown` runs
* `tearDownAfterClass` runs
* Class is done
* Next class loads